webpack-build.config.js

```diff
@@ -27,12 +17,23 @@ webpackConfig.plugins.push(
 
   // Reference: http://webpack.github.io/docs/list-of-plugins.html#uglifyjsplugin
   // Minify all javascript, switch loaders to minimizing mode
-  new webpack.optimize.UglifyJsPlugin({
+  new webpack.optimize.UglifyJsPlugin( {
     compress: {
       warnings: false
     }
-  })
-
-);
+  } )
+];
+
+webpackConfig.plugins.push.apply( webpackConfig.plugins, webpackConfigPlugins );
```

Comment:
No 5 is stable, please read guides from nodejs https://nodejs.org/en/blog/community/node-v5/

As it's not production running code let's keep it up to date.